### PR TITLE
Add tests for SliderImagesDesignable #no-public-changes

### PR DESCRIPTION
**Pull request by SD10**
### Summary of Pull Request:
- Adds tests for `SliderImagesDesignable` protocol
- Simplifies some of the protocols if-else logic to use the nil coalescing operator

### CODE REVIEW

File: IBAnimatableTests/SliderImagesDesignableTests.swift [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+//  IBAnimatable
+//
+//  Created by Steven Deutsch on 5/5/17.
+//  Copyright © 2017 IBAnimatable. All rights reserved.
+//
+
+import XCTest
+@testable import IBAnimatable
+
+class SliderImagesDesignableTests: XCTestCase {
```
Comment (tbaranes) [inline IBAnimatableTests/SliderImagesDesignableTests.swift]: final 😛

**Review by tbaranes — APPROVED** [review]

File: IBAnimatableTests/SliderImagesDesignableTests.swift (hunk 1 above)
Comment (SD10) [reply IBAnimatableTests/SliderImagesDesignableTests.swift]: 😭 dang it!

File: IBAnimatableTests/AnimatableSliderTests.swift [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    animatableSlider = AnimatableSlider()
+  }
+
+  override func tearDown() {
+    super.tearDown()
+  }
+
+}
+
+// MARK: - AnimatableSlider Tests
```
Comment (SD10) [inline IBAnimatableTests/AnimatableSliderTests.swift:27]: Should be `SliderImagesDesignable` Tests but will update in another PR

**Comment by SD10** [comment]
#434 - marking this for record keeping

**Pull request merged**